data/hulls/ship_data.csv

```diff
@@ -7,50 +7,50 @@ Arbitrator,diableavionics_arbitrator,,Diable Avionics,diableavionics_booster,,30
 Arbitrator Z,diableavionics_arbitratorZ,,Diable Avionics,,,350,75,400,,150,0,,200,150,75,90,180,10,NONE,,0,0,0,,,2,2,,,,,,,,,,,,,,,,,,,,100
 ,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
 Frost,diableavionics_frost,,Diable Avionics,diableavionics_slash,,800,100,500,,150,0,,200,250,250,90,180,50,NONE,,,,,,,3,3,,,,,,,,,,,,,,,,,,,,6
-Strife,diableavionics_strife,,Diable Avionics,diableavionics_dakka,,900,125,750,,200,0,,180,180,180,90,90,55,NONE,,,,,,,5,5,,,,,,,,,,,,,,,,,,,,8
-Hoar,diableavionics_hoar,,Diable Avionics,diableavionics_kata,,750,75,750,,200,0,,250,320,240,90,180,55,NONE,,,,,,,4,4,,,,,,,,,,,,,,,,,,,,8
+Strife,diableavionics_strife,,Diable Avionics,diableavionics_dakka,,1300,175,750,,200,0,,180,180,180,90,90,55,NONE,,,,,,,5,5,,,,,,,,,,,,,,,,,,,,8
+Hoar,diableavionics_hoar,,Diable Avionics,diableavionics_kata,,1000,75,750,,200,0,,250,320,240,90,180,55,NONE,,,,,,,4,4,,,,,,,,,,,,,,,,,,,,8
 ,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
 Warlust,diableavionics_warlust,,Diable Avionics,diableavionics_warlust_drone,,1200,125,500,,150,0,,150,150,150,90,180,60,NONE,,,,,,,4,4,,,,,,,,,,,,,,,,,,,,7
 Avalanche,diableavionics_avalanche,,Diable Avionics,diableavionics_berserk,,1500,200,500,,150,0,,160,175,100,50,100,55,NONE,,,,,,,4,4,,,,,,,,,,,,,,,,,,,,7
 Blizzaia,diableavionics_blizzaia,,Diable Avionics,flarelauncher_active,,1600,150,500,,100,0,,155,150,75,50,75,70,NONE,,,,,,,6,6,,,,,,,,,,,,,,,,,,,,11
 ,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
 Valiant,diableavionics_valiant,,Diable Avionics,diableavionics_transit,,800,75,1200,,200,0,,160,200,200,90,180,45,OMNI,,120,0.5,1,,,3,3,,,,,,,,,,,,,,,,,,,,9
-Raven,diableavionics_raven,,Diable Avionics,diableavionics_flicker,,2000,225,1250,,250,0,,120,120,80,50,50,75,NONE,,,,,,,6,6,,,,,,,,,,,,,,,,,,,,10
-Zephyr,diableavionics_zephyr,,Diable Avionics,diableavionics_splitter,,1500,175,750,,200,0,,165,240,150,75,150,65,NONE,,,,,,,5,5,,,,,,,,,,,,,,,,,,,,9
+Raven,diableavionics_raven,,Diable Avionics,diableavionics_flicker,,3000,300,1250,,250,0,,120,120,80,50,50,75,NONE,,,,,,,6,6,,,,,,,,,,,,,,,,,,,,10
+Zephyr,diableavionics_zephyr,,Diable Avionics,diableavionics_splitter,,1400,100,2000,,300,0,,180,240,150,75,150,65,OMNI,,240,0.5,1,,,5,5,,,,,,,,,,,,,,,,,,,,9
```

Comment:
Raven hull increased to 3000 (from 2000)
Zephyr hull decreased to 1400 (from 1500), armor decreased to 100 (from 175), flux capacity increased to 2000 (from 750), flux dissipation increased to 300 (from 200), base speed increased to 180 (from 165), now has an omni shield with arc 240, 0.5 upkeep (150 flux/s), and 1 shield efficiency

Reply:
Zephyr also has Point Defense AI built-in now